packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Modify error text in SyntaxValidator.invalidOptionError. [#2138](https://github.com/zowe/zowe-cli/issues/2138)
```

Comment:
Sorry for being picky, but whenever there are text-only changes I think we go with a patch/fix category.

The original issue if labeled as an enhancement so I can understand why we say it here too.

But curious about when we backport this to the `master` branch...
Do we plan to publish this as a `release-patch` or `release-minor` ? 🤔

Reply:
release-patch!